### DISCUSSION TITLE
Cutting off generated tests

### DISCUSSION THREAD

**Discussion by sherif816** [arkypita/LaserGRBL#2207] [Ideas]
Hi, it would be very nice if I could set to cutting off all type of the generated test (btw awesome feature!!!!) so i can store it for future. Then i will just look to my "material archiv", take out the right type of material and look which setting i want. Now i must make test and than use some rectangle with right size/settings and set it to cut.

**Comment by arkypita** [comment]
Nice idea, but you should know in advance power and speed for successful cut!?

**Reply by sherif816** [reply]
So let it on user to set, for most materials I know, or I can estimate what setting will cut throw the material
